data/crac/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/json/RoundTripUtil.java

```diff
@@ -46,6 +58,14 @@ private RoundTripUtil() {
             objectMapper.registerModule(new Jdk8Module());
             objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
             ObjectWriter writer = objectMapper.writerWithDefaultPrettyPrinter();
+            SimpleModule module = new SimpleModule();
+            module.addSerializer(FreeToUse.class, new FreeToUseSerializer());
+            module.addSerializer(OnConstraint.class, new OnConstraintSerializer());
+            module.addSerializer(OnContingency.class, new OnContingencySerializer());
+            module.addSerializer(PstWithRange.class, new PstWithRangeSerializer());
+            module.addSerializer(AlignedRangeAction.class, new AlignedRangeActionSerializer());
+            module.addSerializer(ComplexNetworkAction.class, new ComplexNetworkActionSerializer());
+            objectMapper.registerModule(module);
```

Comment:
We could put all this in crac-impl, in a CracImplJsonModule, which extends SimpleModule.
And here only do `objectMapper.registerModule(new CracImpJsonModule());`

Reply:
:ok_woman: